src/lpc176x/usbserial.c

```diff
@@ -16,6 +16,7 @@
 #include "internal.h" // gpio_peripheral
 #include "sched.h" // DECL_INIT
 #include "usb_cdc_ep.h" // USB_CDC_EP_BULK_IN
+#include "uid.h" // platform_get_uid
 
```

Comment:
I'd say move `usb_set_serial()`, uid.h, and uid.c all into src/lpc176x/usbserial.c.

Reply:
Sounds good and done. When I start adding other platforms usb_set_serial, or at least the part of the function that breaks a byte array into hex string may need to be accessible by other platforms' code. We can save that for future pull request.